packages/utilities/psammead-assets/src/svgs.jsx

```diff
@@ -1,4 +1,5 @@
 export BBC_BLOCKS from './svgs/bbcBlocks';
+export icons from './svgs/mediaIcons';
```

Comment:
For consistency in naming, we should either call these `mediaIcons` or `icons` everywhere.